imports/api/units.js

```diff
@@ -94,13 +94,19 @@ if (Meteor.isServer) {
           factory.publishById({ // It would work exactly the same for the name according to the BZ API docs
             uriTemplate: ids => {
               const idsQueryParams = ids.map(id => `ids=${id}&`).join('')
-              return `/rest/product?${idsQueryParams}&include_fields=${['name,id'].concat(additionalFields).join(',')}`
+              return `/rest/product?${idsQueryParams}&include_fields=${['name,id,is_active'].concat(additionalFields).join(',')}`
             }
           }),
           withMetaData({
             bzId: 1,
             displayName: 1,
-            moreInfo: 1
+            moreInfo: 1,
+            ownerIds: {
+              $elemMatch: {
+                $in: [this.userId]
+              }
+            },
+            disabled: true
```

Comment:
Although it has the same effect, please change `true` to `1` for consistency.

Reply:
fixed